src/main/java/io/curity/authenticator/netid/endpoints/authenticate/EnterUserNameRequestHandler.java

```diff
@@ -285,6 +285,9 @@ private AuthenticateResponse authenticate(@Nullable String personalNumber, boole
                 case ALREADY_IN_PROGRESS:
                     _logger.trace("Call to {} service failed as authentication is already in progress",
                             serviceName);
+                case USER_BLOCKED:
+                    _logger.trace("Call to {} service failed as authentication has been temporarily blocked due to multiple requests for the same account",
```

Comment:
Use debug, as this would interesting for an admin.

Reply:
Fixed